### PR TITLE
Show security screen when app is inactive.

### LINKED ISSUE #139: Security screen
Like with the current app, we should hide the context switcher preview with the splash screen when the app is in the background

### PR DESCRIPTION
**Pull request by matthewcarlreetz**
Closes #139

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -10,6 +9,8 @@ import {
   UIManager,
   LogBox,
 } from 'react-native'
+import { ThemeProvider } from '@shopify/restyle'
+
```
Comment (tyler-whitman) [inline src/App.tsx:13]: Nit picking, but we can remove the space haha

File: src/features/security/SecurityScreen.tsx [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+type Props = { visible: boolean }
+const SecurityScreen = ({ visible }: Props) => {
+  const fadeAnim = useRef(new Animated.Value(0))
+  const [mounted, setMounted] = useState(false)
+
+  const fadeIn = () => {
+    setMounted(true)
+    Animated.timing(fadeAnim.current, {
+      toValue: 1,
+      duration: 500,
```
Comment (tyler-whitman) [inline src/features/security/SecurityScreen.tsx:15]: I think this needs to be a little faster. Sometimes when I minimize on iOS I can still see the content and the security screen is slightly faded above it. When I set it to `100` it seems to work

Comment (tyler-whitman) [reply src/features/security/SecurityScreen.tsx:15]: I think whats happening is maybe the animation stops as it goes to background?

Comment (tyler-whitman) [reply src/features/security/SecurityScreen.tsx:15]: Hmm it also might be the map slowing down the animation thread? I think it happens on when I am also still loading the map. Might be a debug only issue

Comment (tyler-whitman) [reply src/features/security/SecurityScreen.tsx:15]: Hmm also doesnt seem to be working on Android. But the visible boolean is getting set correctly. My android device is pretty slow though.. but I'm not seeing it ever appear

Comment (matthewcarlreetz) [reply src/features/security/SecurityScreen.tsx:15]: You're right about Android. It appears to not be working in the old app either. I'll investigate.

**Review by tyler-whitman — APPROVED** [review]
Nice, working well on Android now too

File: src/features/security/SecurityScreen.tsx (hunk 2 above)
Comment (tyler-whitman) [reply src/features/security/SecurityScreen.tsx:15]: Fixed with your new changes

**Pull request merged**